src/connector.rs

```diff
@@ -607,4 +629,33 @@ impl EthConnectorContract {
     fn check_used_event(&self, key: &str) -> bool {
         sdk::storage_has_key(&self.used_event_key(key))
     }
+
+    /// Get Eth connector paused flags
+    pub fn get_paused_flags(&self) {
+        let data = self.get_paused().try_to_vec().unwrap();
```

Comment:
This should be an expect with `ERR_FAILED_PARSE`
```suggestion
        let data = self.get_paused().try_to_vec().expect(ERR_FAILED_PARSE);
```